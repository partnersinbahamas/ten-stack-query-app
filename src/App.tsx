import './App.module.scss';

function App() {
  return (
    <section>
      <h1>TSQAPP</h1>
    </section>
  );
}

export default App;