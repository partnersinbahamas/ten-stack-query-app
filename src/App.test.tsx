import { render, screen } from '@testing-library/react';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
import App from './App';

test('renders learn react link', () => {
  const { container } = render(
    <App />,
    { wrapper: () => <QueryClientProvider client={new QueryClient()} />}
  );
  expect(container).toMatchSnapshot();
});
