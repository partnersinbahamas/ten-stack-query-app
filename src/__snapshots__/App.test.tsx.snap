// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <section>
    <h1>
      TSQAPP
    </h1>
  </section>
</div>
`;
